packages/web-components/fast-components/src/tooltip/tooltip.stories.ts

```diff
@@ -11,9 +11,28 @@ FASTDesignSystemProvider;
 addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
     if (name.toLowerCase().startsWith("tooltip")) {
         connectAnchors();
+        const showButton: HTMLElement | null = document.getElementById("anchor-show");
+        if (showButton !== null) {
+            showButton.addEventListener("click", onShowClick);
+        }
     }
 });
 
+function onShowClick(e: MouseEvent): void {
+    toggleTooltipVisibility("tooltip-show-1");
+    toggleTooltipVisibility("tooltip-show-2");
+    toggleTooltipVisibility("tooltip-show-3");
+    toggleTooltipVisibility("tooltip-show-4");
+}
+
+function toggleTooltipVisibility(tooltipId: string): void {
+    const tooltipInstance: HTMLElement | null = document.getElementById(tooltipId);
```

Comment:
[Curious] Could've we use `querySelector` here to clean up the type castings?
```suggestion
    const tooltipInstance: FASTTooltip | null = document.querySelector(`#${tooltipId}`);
```
and a bit down
```ts
tooltipInstance.visible = !tooltipInstance.visible;
```

Reply:
I think `getElementById` is fine, but the types can be asserted instead of declared:

```suggestion
    const tooltipInstance = document.getElementById(tooltipId) as FASTTooltip | null;
```

Reply:
updated